packages/manager/modules/server-sidebar/src/controller.js

```diff
@@ -363,9 +363,11 @@ export default class OvhManagerServerSidebarController {
             }
 
             each(
-              items.sort(({ displayName: a }, { displayName: b }) =>
-                a.localeCompare(b),
-              ),
+              get(typeServices, 'type.sort') === false // If the sorting is already done on the 2 api side
```

Comment:
To avoid `lodash.get`

```suggestion
              typeServices?.type?.sort === false // If the sorting is already done on the 2 api side
```

Reply:
This is to keep it consistent with the rest of the component

Reply:
As discuss together i understand your point of view.
So good for me 👍